components/carousel-indicators/CarouselIndicators.tsx

```diff
@@ -0,0 +1,16 @@
+import clsx from 'clsx'
+import { View } from 'react-native'
+
+interface Props {
+  getPager: number
```

Comment:
usar un nombre mas declarativo, en su lugar : 
```suggestion
  viewPosition: number
  key: number
```